wandb/errors/term.py

```diff
@@ -1,13 +1,29 @@
 import click
+import os
 import sys
 
+from wandb import util
+
 
 LOG_STRING = click.style('wandb', fg='blue', bold=True)
 LOG_STRING_NOCOLOR = 'wandb'
 ERROR_STRING = click.style('ERROR', bg='red', fg='green')
 WARN_STRING = click.style('WARNING', fg='yellow')
 PRINTED_MESSAGES = set()
 
+_silent = False
+_show_info = True
+_show_warnings = True
+_show_errors = True
+
+
+def termsetup(settings):
+    global _silent, _show_info, _show_warnings, _show_errors
+    _silent = not settings._show_console
```

Comment:
show_console is confusing, maybe we should just have this be "silent" in settings also?